public/emoji/activity.css

```diff
@@ -0,0 +1,1933 @@
+            background-position: -409px -477px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3b3 {
+            background-position: -443px -477px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3b2 {
+            background-position: -477px -477px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3b1 {
+            background-position: -511px -1px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3b0 {
+            background-position: -511px -35px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3af {
+            background-position: -511px -69px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3ae {
+            background-position: -511px -103px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3ad {
+            background-position: -511px -137px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3ac {
+            background-position: -511px -171px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3ab {
+            background-position: -511px -205px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3aa {
+            background-position: -511px -239px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3a8 {
+            background-position: -511px -273px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3a7 {
+            background-position: -511px -307px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3a4 {
+            background-position: -511px -341px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f3a3 {
+            background-position: -511px -375px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f39f {
+            background-position: -511px -409px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f397 {
+            background-position: -511px -443px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        .em-activity-1f396 {
+            background-position: -511px -477px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        @media screen and (-webkit-min-device-pixel-ratio: 1.0), screen and (min--moz-device-pixel-ratio: 1.0), screen and (-o-min-device-pixel-ratio: 100/100), screen and (min-device-pixel-ratio: 1.0), screen and (min-resolution: 1.0dppx) {
+            .em-activity-26f9,
```

Comment:
Same here.

Reply:
This is a generated file :slightly_smiling_face: